src/server/array_func.c

```diff
@@ -893,6 +895,16 @@ create_subjob(job *parent, char *newjid, int *rc)
 		return NULL;
 	}
 
+	psub = &subj->ji_wattr[JOB_ATR_outpath];
+	strcpy(tmp_path, psub->at_val.at_str);
+	job_attr_def[JOB_ATR_outpath].at_decode(psub, NULL, NULL,
+		subst_array_index(subj,tmp_path));
```

Comment:
leave a space after the comma

Reply:
Fixed in new commit rc2 https://github.com/PBSPro/pbspro/pull/892/commits/90b8b4df18442ed3f8016032e9c1cc43f83cdf5f